src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```diff
@@ -713,9 +713,15 @@ protected function buildDeployment(Device $deviceForFunctions, Device $deviceFor
                     ->addOption(new XStatic());
                 $detection = $detector->detect();
 
-                $resource->setAttribute('adapter', $detection->getName());
-                $resource->setAttribute('fallbackFile', $detection->getFallbackFile() ?? '');
-                $resource = $dbForProject->updateDocument('sites', $resource->getId(), $resource);
+                $adapter = $resource->getAttribute('adapter', null);
+
+                if ($adapter === null) {
```

Comment:
Let's follow pattern as before, `empty($resource->getAttribute('adapter', ''))`
> empty() check, and empty string default

Reply:
Updated. Using a variable as we need it in a few places.